docker-compose.minimal.yml

```diff
@@ -399,6 +399,7 @@ services:
     environment:
       - FLAGD_HOST
       - PRODUCT_CATALOG_PORT
+      - PRODUCT_CATALOG_RELOAD_INTERVAL
```

Comment:
Should we mount the products to the container?

Reply:
products.json is mounted in the minimal config now